### PR TITLE
Task Manager

### PR DESCRIPTION
**Pull request by Shiveshtrivedi**
Adding date when task is created - done
Sorting functionality - done
CSS of box  - done
Remove alert and show message on popup window - done
Naming convention - done
Document formatted by prettier - done
Remove console - done
Remove icon and use Unicode - done
Used arrow function only - done

### CODE REVIEW

File: task_management/data_file.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const url = "https://66a0bb4b7053166bcabc8698.mockapi.io/Data";
```
Comment (aakash-soni1) [inline task_management/data_file.js:1]: Give spacing in between new function  and variables

File: task_management/index.css [hunk 2]
```diff
@@ -0,0 +1,323 @@
+body {
```
Comment (aakash-soni1) [inline task_management/index.css:1]: Don't use rem and em use value in px

File: task_management/index.css [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+.popupOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
```
Comment (aakash-soni1) [inline task_management/index.css]: why you have used z-index 1000

File: task_management/index.css [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+  margin-top: 3%;
+}
+.date {
+  font-style: italic;
+  font-size: 70%;
+  color: #2b15ec;
+  margin: 5% 10% 0 0;
+}
+
+@media (max-width: 480px) {
```
Comment (aakash-soni1) [inline task_management/index.css:253]: put media queries at the top of file

File: task_management/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (aakash-soni1) [inline task_management/index.html]: Give the proper Title

**Review by aakash-soni1 — CHANGES_REQUESTED** [review]

File: task_management/index.css (hunk 3 above)
Comment (Shiveshtrivedi) [reply task_management/index.css:75]: Just wanted that popup window visible over whole screen

File: package.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "@eslint/js": "^9.8.0",
+    "eslint": "^9.8.0",
+    "eslint-plugin-react": "^7.35.0",
+    "globals": "^15.8.0",
```
Comment (aakash-soni1) [inline package.json]: Why have you use global as dependencies ?

File: task_management/index.js [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+  }
+};
+
+const sortTasksByName = (ascending = true) => {
+  allTasks.sort((a, b) => {
+    const nameA = a.name.toLowerCase();
+    const nameB = b.name.toLowerCase();
+
+    if (nameA < nameB) return ascending ? -1 : 1;
+    if (nameA > nameB) return ascending ? 1 : -1;
```
Comment (aakash-soni1) [inline task_management/index.js]: Why two conditions are there for same things, does the two condition is doing same thing or different ?

Comment (aakash-soni1) [reply task_management/index.js]: find a optimize way to do this logic

**Review by aakash-soni1 — CHANGES_REQUESTED** [review]

**Pull request merged**